### ISSUE TITLE
508-defect-3 [SCREENREADER, COGNITION]: Phone numbers SHOULD use the new aria-label pattern

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9299]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

By default, numbers may be read in a variety of ways by different screen readers. It may be challenging to understand the numbers. There is a new pattern for phone numbers on VA.gov, that adds an `aria-label="8 0 0. 5 5 5. 1 2 3 4."` attribute to the phone number link. This pattern **should** be added to all phone numbers throughout the application.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to understand the phone number I should call.

## Environment

* Operating System: all
* Browser: any
* Screenreading device: all
* Server destination: staging

## Steps to Recreate

1. Enter https://staging.va.gov/caregivers` in browser [2020-07-28 new url: https://staging.va.gov/family-member-benefits/apply-for-caregiver-assistance-form-10-10cg/introduction]
2. Start screenreading device listed in Environment
3. Navigate the page content
4. Verify that the phone number may be hard to understand

Note: numbers are read differently by different screen readers; numbers may be read as hundreds or thousands, very fast, or without pauses

## Possible Fixes (optional)

Adds an `aria-label="8 0 0. 5 5 5. 1 2 3 4."` attribute to the phone number link, as documented in [this PR](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/260) and [this issue ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9114).

## WCAG or Vendor Guidance (optional)

* [I thought title text improved accessibility. I was wrong.](https://silktide.com/blog/2013/i-thought-title-text-improved-accessibility-i-was-wrong)

* [Accessibility and phone number formatting](https://thatdevgirl.com/blog/accessibility-phone-number-formatting)

## Screenshots or Trace Logs

![Screen Shot 2020-05-19 at 3 45 31 PM](https://user-images.githubusercontent.com/57469/82374859-a7d0cf80-99ed-11ea-92eb-d527fcef9082.png)

**Comment by jonathanjnelson** [comment]
@jenstrickland Feedback was integrated into staging environment. Can you confirm and close?

**Comment by jenstrickland** [comment]
On the introduction page: `Contact the National Caregiver Support Line at855-260-3274or aVeterans Service Organizationto get help filling out the form` does not use the aria-label pattern. (This text was copied from the page, and there aren't spaces between words and the links

**Comment by LinnJS** [comment]
@jenstrickland fix is in staging should be resolved and ready to be validated.

**Comment by jenstrickland** [comment]
@LinnJS Validated. 🎉
